### PR TITLE
Additional data for analysis chart

### PR DESCRIPTION
**Pull request by danielfdsilva**
- Adds the dataset description to the chart card info tooltip and a button to go to the dataset overview pag
- Includes the `median` value as a chart option

### CODE REVIEW

**Comment by danielfdsilva** [comment]
@ricardoduplos Including an additional value in the legend breaks things a bit on small screen. Do you have ideas for a more scalable legend?

File: app/scripts/styles/theme.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ export default function themeOverrides() {
       infographicA: '#fcab10',
       infographicB: '#f4442e',
       infographicC: '#b62b6e',
-      infographicD: '#2ca58d'
+      infographicD: '#2ca58d',
+      infographicE: '#2276ac'
```
Comment (danielfdsilva) [inline app/scripts/styles/theme.js:14]: @ricardoduplos Can you provide more infographic colors? Or should we use a generator function?

**Comment by hanbyul-here** [comment]
Any reason this is still a draft?

**Comment by danielfdsilva** [comment]
@hanbyul-here Need finishing design touched from @ricardoduplos

**Review by ricardoduplos — APPROVED** [review]

**Comment by danielfdsilva** [comment]
@hanbyul-here Ready to check! :)

**Review by hanbyul-here — APPROVED** [review]
Nice, especially the button inside of the info tooltip is keyboard navigable. 
(+ not related to this pr but realized that 1. our export options are not keyboard navigatable, 2. YAxis label is not getting displayed properly on small screens) we can handle it separately.)

**Pull request merged**